middlewares/authenticate.js

```diff
@@ -0,0 +1,29 @@
+const jwt = require("jsonwebtoken");
+const User = require("../models/schemas/users");
+const { HttpError } = require("../helpers");
+const { SECRET_KEY } = process.env;
+
+const authenticate = async (req, res, next) => {
+  const { authorization = "" } = req.headers;
+  const [bearer, token] = authorization.split(" ");
+
+  if (bearer !== "Bearer") {
+    next(new HttpError(401, "Not authorized"));
```

Comment:
```suggestion
    return next(new HttpError(401, "Not authorized"));
```
краще додавати return, коли функція має припинити виконання. Іноді це упущення призводить до помилок